packages/component-library/src/components/form/mt-link/mt-link.interactive.stories.ts

```diff
@@ -0,0 +1,44 @@
+import { within } from "@storybook/test";
+import { expect } from "@storybook/test";
+
+import meta, { type MtLinkMeta, type MtLinkStory } from "./mt-link.stories";
+
+export default {
+  ...meta,
+  title: "Interaction Tests/Form/mt-link",
```

Comment:
It makes more sense to move the both story files unter `Navigaton/mt-link`

Reply:
Thanks, @Haberkamp , I moved it to `Navigation/mt-link`.